assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -161,6 +161,16 @@ che.docker.always_pull_image=true
 # to be able to launch their own runtimes which are embedded Docker containers.
 che.docker.privileged=false
 
+# This parameter allows to specify custom security options for the created docker container.
+# seccomp:unconfined is the default for kubernetes, but not for docker. This is needed
+# for debugging with gdbserver. See https://github.com/eclipse/che/issues/4284 for details.
+# Parameters are passed as an array, so you can add multiple 
+# values comma seperated. Please also see https://docs.docker.com/engine/api/v1.21/#2-endpoints,
+# in particular the "HostConfig":{"SecurityOpt": []} entry. If this parameter is empty,
+# docker blocks certain Syscalls by default https://docs.docker.com/engine/security/seccomp/
+# WARNING: if you give incorrect values, docker gives an error and doesn't start the containre
```

Comment:
Here is a little typo: `containre` -> `container`. The same typo is in che.env

Reply:
fixed